packages/clerk-js/src/utils/cookies/devBrowser.ts

```diff
@@ -0,0 +1,5 @@
+import { createCookieHandler } from '@clerk/shared';
+
+const DEV_BROWSER_COOKIE_NAME = '__clerk_db_jwt';
```

Comment:
Can we try not redeclaring the constant here? 
Here's another place in the code that defines the same value: https://github.com/clerkinc/javascript/blob/d53432e4a57df64cd0556d53ff20eb0a23f4d3f5/packages/clerk-js/src/utils/devBrowser.ts#L3